lambda/src/new_account_support_case.py

```diff
@@ -25,6 +25,11 @@
     datefmt="%Y-%m-%dT%H:%M:%S",
 )
 
+LOCALSTACK_IP = os.getenv("LOCALSTACK_HOSTNAME")
+ORG_ENDPOINT = "http://localstack:4615" if LOCALSTACK_IP else None
```

Comment:
For these endpoints, instead of hardcoding `localstack` in the lambda, would it make sense to just use the `LOCALSTACK_IP` value? Pretty sure this is the primary reason localstack injects `LOCALSTACK_HOSTNAME` into the lambda environment, and it would be a little more resilient to changes in how localstack and docker are setup.

```suggestion
ORG_ENDPOINT = f"http://${LOCALSTACK_IP}:4615" if LOCALSTACK_IP else None
```

Reply:
I could use the IP, but "localstack" is hardcoded in several places (docker-compose file, Makefile, and integration test (`localstack-client`), so at least it's consistent.  I had other versions of the above at different times.  I will change it, but it will be to use the LOCALSTACK_HOSTNAME as a boolean and not as a value.

Reply:
Ok, it is not actually an IP, though. Sometimes it resolves to the name, sometimes the IP. Just depends on the user's setup.